rpmlint/descriptions/BinariesCheck.toml

```diff
@@ -12,6 +12,9 @@ noarch-with-lib64="""
 This package is marked as noarch but installs files into lib64.
 Not all architectures have this in path, so the package can't be noarch.
 """
+no-soname="""
+The library has no soname.
```

Comment:
This should be `no-soversion`. Shared object names do not require versions. You're attempting to mandate that a soname contains a soversion. Adjust accordingly.

Reply:
Alternatively, it can be titled `no-version-in-soname`.

Reply:
I don't think so. This 'no-soname' check tests the library for an occurrence of the SONAME entry in the dynamic section (readelf -d <library>). 
Maybe we should just extend the description to be more verbose (something like: "The library has no SONAME entry in its dynamic section.")? But I'm not sure if it will help users or if it confuses them even more.

Reply:
Hmm, we probably should have that check. That description isn't terrible, but it's not too helpful either (what's a dynamic section?).

Reply:
Yep, it's too complicated for the user. I would vote for leaving it as is while the user can find more information in the wiki (e.g. https://fedoraproject.org/wiki/Common_Rpmlint_issues#no-soname)